tests/testthat/test-check_dimensions.R

```diff
@@ -1,50 +1,48 @@
 
-ngm_fairfield_85_plus <- generate_ngm(
-  lga_name = "Fairfield (C)",
-  age_breaks = c(seq(0, 85, by = 5), Inf),
-  R_target = 1.5
-)
 
-ngm_fairfield_80_plus <- generate_ngm(
-  lga_name = "Fairfield (C)",
-  age_breaks = c(seq(0, 80, by = 5), Inf),
-  R_target = 1.5
-)
 
 
+test_that("check_dimensions() returns nothing when compatible dimensions",
+          {
+            list(matrix(1:6, nrow = 3, ncol = 2),
+                 matrix(1:6, nrow = 3, ncol = 2)) -> demo_matrix
+            
+            names(demo_matrix) <- c("matrix_a", "matrix_b")
+            
+            demo_data <- tibble::tibble(x = 1:2, y = 2 * x)
+            
+            expect_silent(check_dimensions(demo_matrix,
+                                           demo_data))
+          })
 
-test_that("check_dimensions() returns nothing when compatible dimensions", {
 
-  
-
-  
-  expect_silent(check_dimensions(ngm_fairfield_80_plus,
-                                         vaccination_effect_example_data))
-})
 
 test_that("check_dimensions() returns error", {
+  list(matrix(1:6, nrow = 3, ncol = 2),
+       matrix(1:6, nrow = 3, ncol = 2)) -> demo_matrix
   
-
-  expect_snapshot_error(check_dimensions(ngm_fairfield_85_plus,
-                                  vaccination_effect_example_data))
-})
-
-test_that("apply_vaccination gives error when incompatible dimensions present", {
-
- expect_snapshot_error(apply_vaccination(
-    ngm = ngm_fairfield_85_plus,
-    data = vaccination_effect_example_data,
-    coverage_col = coverage,
-    acquisition_col = acquisition,
-    transmission_col = transmission
-  ))
+  names(demo_matrix) <- c("matrix_a", "matrix_b")
   
+  demo_data <- tibble::tibble(x = 1:6, y = 2 * x)
+  expect_snapshot_error(check_dimensions(demo_matrix,
+                                         demo_data))
 })
 
-# ngm_nsw_vacc <- apply_vaccination(
-#   ngm = ngm_fairfield_80_plus,
-#   data = vaccination_effect_example_data,
-#   coverage_col = coverage,
-#   acquisition_col = acquisition,
-#   transmission_col = transmission
-# )
\ No newline at end of file
+test_that("apply_vaccination gives error when incompatible dimensions present",
+          {
+            ngm_fairfield_15_plus <- generate_ngm(
```

Comment:
You can probably swap this out for a dummy list as in the `test_check_dimensions` above, so you don't need to run `generate_ngm`, which takes a little while